src/Core/Components/Switch/FluentSwitch.razor.css

```diff
@@ -0,0 +1,3 @@
+fluent-field[readonly]:has(fluent-switch) {
+  pointer-events: none;
```

Comment:
This style must be applied to µ fluent-switch` and not `fluent-field` => `fluent-switch[readonly] { pointer-events: none; }`

Reply:
ok, done but it doesn't have the expected behaviour in the example.